### DISCUSSION TITLE
How do I make the anchor link work?

### DISCUSSION THREAD

**Discussion by ershovg** [idiotWu/smooth-scrollbar#337] [Q&A] [answered]
## Environment
- Browser: Chrome
- Version of smooth-scrollbar: last

how to make anchor links? they cannot scroll inside a 100vh container. I need to link to #id sections on my site. How do I make the anchor link work?

**Comment by ershovg** [comment]
I found a solution using Jquery + the ScrollTo method, however I don't like the fact that I need to create a lot of functions to make 2 or more of my links work. Tell me how to do it correctly and reduce the code? A demonstration of the code.

https://codepen.io/GeorgeDesign2020/full/bGeWNOE

https://codepen.io/GeorgeDesign2020/pen/bGeWNOE

**Comment by dolphin-wood** [comment, marked as answer]
See #128 and https://jsbin.com/jequfuq/edit
